ansible/templates/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3.3"
+
+services:
+
+  db:
+    image: postgres:12-alpine
+    restart: always
+    environment:
+      - POSTGRES_USER=lemmy
+      - POSTGRES_PASSWORD=${DATABASE_PASSWORD}
+      - POSTGRES_DB=lemmy
+    volumes:
+      - ./volumes/db:/var/lib/postgresql/data
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U lemmy"]
+      interval: 5s
+      timeout: 5s
+      retries: 20
+
+  lemmy:
+    image: dessalines/lemmy:v0.0.7
```

Comment:
Seems like it might be a good idea to extract the version number into the .env also, if that's possible.

Reply:
Thats not usually done, the point is to use .env for passwords and instance-specific stuff, while docker-compose.yml can be copied straight from git (and so contain version numbers etc). Thats not how it works here, but seems to be the idea. Eg someone could just wget docker-compose when a new update is out.

Reply:
Ah that makes sense, gotcha.